app/konnect/analytics/index.md

```diff
@@ -3,89 +3,36 @@ title: Introduction to Monitoring Health with Analytics
-
-* **Traffic**
-* **Errors**
-* **Latency**
-
-These categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
-You can view a graph for each category by clicking **Traffic**, **Errors**, or **Latency**, and switching between the views.
-
-* **Traffic**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval.
-    * Successful requests contain all requests that returned a **1xx-3xx** status code.
-    * Failed requests contain all requests that returned a **4xx-5xx** status code.
-    * 6xx status codes are not reported.
-
-    ![traffic analytics graph](/assets/images/products/konnect/analytics/konnect-vitals-traffic.png){:.image-border}
-    > _**Figure 2:** Graph showing successful and failed requests over the past three hours._
-
-* **Errors**: This graph displays the total number of failed HTTP requests categorized by error response codes over the specified time interval. Error response codes include any **4xx-5xx** status codes.
-
-    ![errors analytics graph](/assets/images/products/konnect/analytics/konnect-vitals-errors.png){:.image-border}
-    > _**Figure 3:** Graph showing errors by 4xx and 5xx error codes received over the past three hours._
-
-* **Latency**: This graph displays request latency, in milliseconds, of the 99th, 95th, and 50th percentiles.
-Admins can monitor the latency, investigate where delays are noticed, and optimize performance for APIs.
-
-    {:.note}
-    > **Note**: Latency data is only available for requests proxied through a data plane running {{site.base_gateway}} 3.0.0.0 or later.
-
-   ![latency analytics graph](/assets/images/products/konnect/analytics/konnect-analytics-latency.png){:.image-border}
-  > _**Figure 4:** Graph showing latency as a percentage over the past 15 minutes._
+## Analyze API usage and performance data
 
-Graphs can be interacted with, including hovering over chart items to display more details, and filtering options by clicking on items in the **legend**.
+{{site.konnect_short_name}}  Analytics {% konnect_icon analytics %} provides different tools to help you track the performance and behavior of your APIs and data plane nodes. You can use these tools to access key statistics, monitor vitals, and pinpoint anomalies in real time. 
 
-You can select a time period using the **time period** drop-down menu. The intervals aggregate data at different increments of time.
+You can use the following diagram to help you determine which analytics tools are best for your use case:
 
-### Export analytics data
+{% mermaid %}
+flowchart TD
+    A((What is your goal?)) -->B(Quickly understand API usage and performance at a glance)
+    A((What is your goal?)) -->C(In-depth analytics of API data)
```

Comment:
This is more about browsing data to gain access. Any chance we can stick with the original?
> Browse API usage data to gain access to key performance and health statistics.